src/main/resources/edu/cornell/kfs/coa/document/workflow/SubObjectCodeGlobalMaintenanceDocument.xml

```diff
@@ -14,26 +14,25 @@
  See the License for the specific language governing permissions and
  limitations under the License.
 -->
-<data xmlns="ns:workflow" xmlns:fo="http://www.w3.org/1999/XSL/Format" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="ns:workflow resource:WorkflowData">
+<data xmlns="ns:workflow" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="ns:workflow resource:WorkflowData">
   <documentTypes xmlns="ns:workflow/DocumentType" xsi:schemaLocation="ns:workflow/DocumentType resource:DocumentType">
     <documentType>
       <name>GSOB</name>
       <parent>COA</parent>
+      <label>Sub-Object Code Global</label>
```

Comment:
Just to confirm, you want sub object code global to be a child of COA, and Global Account Reversion to be a child of COSM?  I am just confirming because I see Global Account Reversion was previously a child of COA.